keycloak/keycloak-admin-ui
# Realm settings(tokens): Add tokens tab

## Motivation

https://issues.redhat.com/browse/APPDUX-1103

## Verification Steps



1. Go to `Realm settings -> tokens`.
2. Populate form fields.
3. Save and verify data is saved accurately. 

## Checklist:


## Additional Notes
